packages/pn-pa-webapp/src/generated-client/tos-privacy/git_push.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
```

Comment:
Remove this file from generation. Check the .openapi-generator-ignore of others generated clients